packages/babel-plugin-transform-named-capturing-groups-regex/test/fixtures/syntax/.escape-no-unicode-flag/actual.js

```diff
@@ -0,0 +1 @@
+/(?<\u{41}>)\k<\u{41}>/;
```

Comment:
@DmitrySoshnikov I think that `regexp-tree` should throw on this regex, since unicode escapes are only alloed when there is the `u` flag?

Reply:
Yeah, @Golmote added generic support for Unicode to parser (https://github.com/DmitrySoshnikov/regexp-tree/issues/114); we should also check the group names, if they are allowed only with `u` flag (since by itself `\u{41}` can normally be used without `u` flag -- "`u` repeated 41 times").

If you're positive from spec, that it should throw, we should open an issue. Thanks for catching it!

Reply:
I'm pretty sure identity escapes don't throw without the `u` flag.

Reply:
@Golmote, right, however this one is specifically for a capturing group name, like in `(?<\u{41}>)`. I'll need to double-check the spec, but seems, yes, it should throw, and be an invalid name for a group if no `u` flag is passed.

Reply:
Sorry I read too fast 😳